smrf/__init__.py

```diff
@@ -1,38 +1,46 @@
 # -*- coding: utf-8 -*-
-'''
-SMRF - Snow Modeling Resource Framework
-'''
-__version__ = '0.9.1'
+# flake8: noqa
 
 import os
 
-__core_config__ = os.path.abspath(os.path.dirname(__file__) + '/framework/CoreConfig.ini')
-__recipes__ = os.path.abspath(os.path.dirname(__file__) + '/framework/recipes.ini')
-__config_changelog__ = os.path.abspath(os.path.dirname(__file__) + '/framework/changelog.ini')
+from pkg_resources import DistributionNotFound, get_distribution
 
+try:
+    __version__ = get_distribution(__name__).version
+except DistributionNotFound:
+    __version__ = 'unknown'
+__core_config__ = os.path.abspath(
+    os.path.dirname(__file__) + '/framework/CoreConfig.ini')
+__recipes__ = os.path.abspath(os.path.dirname(
+    __file__) + '/framework/recipes.ini')
+__config_changelog__ = os.path.abspath(
+    os.path.dirname(__file__) + '/framework/changelog.ini')
```

Comment:
These are special `inicheck` variables that are read when parsing a config file. While SMRF doesn't use them, inicheck will look for them because of the following line. We could probably make inicheck a bit more flexible or configurable in the future.

https://github.com/USDA-ARS-NWRC/smrf/blob/4e1ebc5df71c77f8919d3c10f74cea842ce23421/smrf/framework/model_framework.py#L109